pytorch_lightning/utilities/io.py

```diff
@@ -0,0 +1,11 @@
+import torch
+
+from urllib.parse import urlparse
+
+
+def load(path_or_url: str, map_location=None):
```

Comment:
we can't really specify this b/c it can be a lot of different things: https://github.com/PyTorchLightning/pytorch-lightning/pull/1505

Welcome any suggestions.